enclave-tls/src/Makefile

```diff
@@ -45,6 +48,9 @@ install:
 	$(INSTALL) -m 755 libtls_wrapper_null.so $(TLS_WRAPPERS_LIBDIR)
 	$(INSTALL) -d -m 755 $(ENCLAVE_QUOTES_LIBDIR)
 	$(INSTALL) -m 755 libenclave_quote_null.so $(ENCLAVE_QUOTES_LIBDIR)
+	cd $(ENCLAVE_TLS_LIBDIR) && mv libenclave_tls.so libenclave_tls.so.$(MAJ_VERSION).$(MIN_VERSION) && \
```

Comment:
Call `$(INSTALL) -m 755 libenclave_tls.so $(ENCLAVE_TLS_LIBDIR)/libenclave_tls.so.$(MAJ_VERSION).$(MIN_VERSION)` directly.

Reply:
This line is not changed.